app/presenters/order_detail_presenter.rb

```diff
@@ -49,6 +53,12 @@ def survey_url
     survey_completed? ? external_service_receiver.show_url : ""
   end
 
+  def wrapped_total
```

Comment:
I traced the `reload` back to 957cbce10f78205802c76c0131c9da84a4d49528.
The `reload` remained when it became`od.reload.wrapped_cost` when introducing `PriceDisplayment` in 0c845be58e61d7c5eac79165131d9397c1b883e6, and in 95e07461bb709de591f6fd6cbb6896ce11b923eb when the method name was renamed to `#wrapped_total`.

It doesn't seem to need the `reload` in testing but maybe it did at one point? I'm removing it.